src/Services/Http/SongServices.ts

```diff
@@ -20,4 +20,14 @@ export class SongServices {
     const url = `station/${stationId}/${youTubeVideoId}`;
     return this._httpServices.post(url, body);
   }
+
+  public upVote(stationId: String, songId: String): Observable<Song> {
```

Comment:
It should be `string` instead of `String`, with `s` in lowercase.
Note that `String` more like Java syntax, you should use `string` with `s` in lower case in TypeScript syntax